src/api/index.spec.tsx

```diff
@@ -91,11 +92,12 @@ describe(__filename, () => {
 
     it('accepts an API version', async () => {
       const version = 'v123';
+      const endpoint = '/';
 
-      await callApiWithDefaultApiState({ endpoint: '/', version });
+      await callApiWithDefaultApiState({ endpoint, version });
 
       expect(fetch).toHaveBeenCalledWith(
-        expect.stringMatching(`/api/${version}/`),
+        expect.stringMatching(makeApiURL({ path: endpoint, version })),
```

Comment:
This probably no longer needs `expect.stringMatching`, right?

Reply:
I kept `stringMatching` because we append `?lang=` to the URL.